src/components/Analysis/Applications/CreateApplicationDialog.vue

```diff
@@ -210,6 +240,36 @@ export default {
           label: "Postprocessor",
         },
       ],
+      cpuItems:[
+        {
+          value: 1024,
+          label: "1 CPU",
+        },
+        {
+          value: 2048,
+          label: "2 CPU",
+        },
+        {
+          value: 4096,
+          label: "4 CPU",
+        },
+        {
+          value: 8192,
+          label: "8 CPU",
+        }
+      ],
+      memoryItems:[],
+      memoryItemMap: new Map([
```

Comment:
should we remove memory values for the ignored fractional vCPU values. There also appears to be some leftover commented code. Could we remove it if it’s unlikely to be needed in the future?

Reply:
I will remove the commented code, but I think the other values will make it easier to map if we want to add other cpu values from the table found on the ticket, in the future. Instead of having to dig back into the table and re-write to the map. But I'm open to removing it as well.

Reply:
sure makes sense. Sry, missed where Annalee mentioned the same.